src/setup_node_env/node_version_validator.js

```diff
@@ -37,15 +37,18 @@ var pkg = require('../../package.json');
 var currentVersion = (process && process.version) || null;
 var rawRequiredVersion = (pkg && pkg.engines && pkg.engines.node) || null;
 var requiredVersion = rawRequiredVersion ? 'v' + rawRequiredVersion : rawRequiredVersion;
-var isVersionValid = !!currentVersion && !!requiredVersion && currentVersion === requiredVersion;
+var requiredVersionMajorMinor = requiredVersion.match(/^v(\d+\.\d+)/)[1];
+var isVersionValid = requiredVersionMajorMinor === currentVersion.match(/^v(\d+\.\d+)/)[1];
 
 // Validates current the NodeJS version compatibility when OpenSearch Dashboards starts.
 if (!isVersionValid) {
   var errorMessage =
-    'OpenSearch Dashboards does not support the current Node.js version ' +
-    currentVersion +
-    '. Please use Node.js ' +
+    'OpenSearch Dashboards was built with ' +
```

Comment:
nit: could use a template literal here too

Reply:
Fixed in 199b8c4d43